modules/images-using-images-s2i-ruby-pulling-images.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/using_images/using-images-source-to-image.adoc
+
+[id="images-using-images-s2i-ruby-pulling-images_{context}"]
+= Pulling images for Ruby
+
+These images come in two options:
+
+* RHEL 8
+* CentOS 7
+
+*RHEL 8 Based Images*
+
+The RHEL 8 images are available through the Red Hat Registry.
+
+.Procedure
+
+* To pull the RHEL 8 image, enter the following command for the version of Ruby you want:
++
+.Ruby `2.0`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/openshift3/ruby-20-rhel7
+----
++
+.Ruby `2.2`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/rhscl/ruby-22-rhel7
+----
++
+.Ruby `2.3`
+[source,terminal]
+----
+$ podman pull registry.redhat.io/rhscl/ruby-23-rhel7
+----
+
+*CentOS 7 Based Images*
+
+These images are available on link:quay.io[Quay.io].
+
+.Procedure
+
+* To pull the CentOS 7 image, enter the following command for the version of Ruby you want:
++
+.Ruby `2.0`
+[source,terminal]
+----
+$ podman pull openshift/ruby-20-centos7
+----
++
+.Ruby `2.2`
+[source,terminal]
+----
+$ podman pull openshift/ruby-22-centos7
+----
++
+.Ruby `2.3`
+[source,terminal]
+----
+$ podman pull centos/ruby-23-centos7
```

Comment:
Above images need updated, registry.redhat.io/rhscl/ruby-25-rhel7:latest, registry.redhat.io/rhscl/ruby-26-rhel7:latest and registry.redhat.io/rhscl/ruby-27-rhel7:latest